Sources/JXKit/JXValue.swift

```diff
@@ -384,25 +397,60 @@ extension JXValue {
 
     /// Returns the JavaScript number value.
     @inlinable public var float: Float {
+        get throws {
+            return try Float(double)
+        }
+    }
+
+    private var doubleForIntegerConversion: Double {
```

Comment:
Getting JXValue.<anyintegertype> now uses this function, so NaN and Infinity now throw.